[BUG] npx prisma generate in github action fails

## Information

- **Version:** 6.116.5
- Packages:
```
{
  "name": "test",
  "version": "1.0.0",
  "license": "UNLICENSED",
  "description": "",
  "scripts": {
    "build": "yarn run barrels && tsc --project tsconfig.compile.json",
    "barrels": "barrelsby --config .barrelsby.json",
    "start": "yarn run barrels && yarn tsnd --inspect --ignore-watch node_modules --respawn --transpile-only -r tsconfig-paths/register src/index.ts",
    "start:prod": "cross-env NODE_ENV=production node dist/index.js",
    "test": "yarn run test:lint && yarn run test:coverage",
    "test:unit": "cross-env NODE_ENV=test jest",
    "test:coverage": "yarn run test:unit",
    "docker:prisma:all": "docker compose run --rm server yarn prisma:all",
    "prisma:all": "yarn prisma:generate && yarn prisma:migrate",
    "prisma:migrate": "npx prisma migrate dev --name init",
    "prisma:generate": "npx prisma generate",
    "prisma:seed": "npx prisma db seed",
    "test:lint": "eslint '**/*.{ts,js}'",
    "test:lint:fix": "eslint '**/*.{ts,js}' --fix",
    "prettier": "prettier '**/*.{ts,js,json,md,yml,yaml}' --write",
    "prepare": "is-ci || husky install"
  },
  "dependencies": {
    "@prisma/client": "^4.0.0",
    "@tsed/ajv": "^6.116.5",
    "@tsed/common": "^6.116.5",
    "@tsed/core": "^6.116.5",
    "@tsed/di": "^6.116.5",
    "@tsed/engines": "^6.116.5",
    "@tsed/exceptions": "^6.116.5",
    "@tsed/json-mapper": "^6.116.5",
    "@tsed/logger": "^6.2.0",
    "@tsed/logger-file": "^6.2.0",
    "@tsed/passport": "^6.116.5",
    "@tsed/platform-cache": "^6.116.5",
    "@tsed/platform-exceptions": "^6.116.5",
    "@tsed/platform-express": "^6.116.5",
    "@tsed/platform-log-middleware": "^6.116.5",
    "@tsed/platform-middlewares": "^6.116.5",
    "@tsed/platform-params": "^6.116.5",
    "@tsed/platform-response-filter": "^6.116.5",
    "@tsed/platform-views": "^6.116.5",
    "@tsed/prisma": "^6.116.5",
    "@tsed/schema": "^6.116.5",
    "@tsed/typegraphql": "^6.116.5",
    "ajv": "^8.11.0",
    "apollo-datasource": "^3.3.1",
    "apollo-datasource-rest": "^3.5.1",
    "apollo-server-core": "^3.6.2",
    "apollo-server-express": "2.25.2",
    "barrelsby": "^2.3.4",
    "body-parser": "^1.20.0",
    "class-validator": "^0.13.2",
    "compression": "^1.7.4",
    "cookie-parser": "^1.4.6",
    "cors": "^2.8.5",
    "cross-env": "^7.0.3",
    "dotenv": "^16.0.1",
    "dotenv-expand": "^8.0.3",
    "dotenv-flow": "^3.2.0",
    "express": "^4.18.1",
    "graphql": "^15.7.2",
    "method-override": "^3.0.0",
    "passport": "^0.6.0",
    "type-graphql": "^1.1.1"
  },
  "devDependencies": {
    "@faker-js/faker": "^7.3.0",
    "@tsed/cli-plugin-eslint": "3.22.4",
    "@tsed/cli-plugin-jest": "3.22.4",
    "@tsed/cli-plugin-passport": "3.22.4",
    "@tsed/cli-plugin-prisma": "3.22.4",
    "@tsed/cli-plugin-typegraphql": "3.22.4",
    "@types/compression": "^1.7.2",
    "@types/cookie-parser": "^1.4.3",
    "@types/cors": "^2.8.12",
    "@types/express": "^4.17.13",
    "@types/jest": "^28.1.4",
    "@types/method-override": "^0.0.32",
    "@types/multer": "^1.4.7",
    "@types/node": "^18.0.3",
    "@types/passport": "^1.0.9",
    "@types/supertest": "^2.0.12",
    "@types/validator": "^13.7.4",
    "@typescript-eslint/eslint-plugin": "^5.30.6",
    "@typescript-eslint/parser": "^5.30.6",
    "apollo-server-testing": "^2.25.3",
    "eslint": "^8.19.0",
    "eslint-config-prettier": "^8.5.0",
    "eslint-plugin-prettier": "^4.2.1",
    "husky": "^8.0.1",
    "is-ci": "^3.0.1",
    "jest": "^28.1.2",
    "lint-staged": "^13.0.3",
    "prettier": "^2.7.1",
    "supertest": "^6.2.4",
    "ts-jest": "^28.0.5",
    "ts-node": "^10.8.2",
    "ts-node-dev": "^2.0.0",
    "tsconfig-paths": "^4.0.0",
    "tslib": "^2.4.0",
    "typescript": "^4.7.4"
  },
  "tsed": {
    "packageManager": "yarn",
    "convention": "default",
    "architecture": "default"
  },
  "prisma": {
    "seed": "ts-node prisma/seed.ts"
  }
}

```

Git hub action:
```
name: Lint and Check

on:
  pull_request:
    paths:
      - .github/workflows/lint-and-check.yml
      - "src/**"
      - ".prisma/**"

concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true

jobs:
  check:
    if: github.event.pull_request.draft == false
    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v2

      - name: Use Node.js
        uses: actions/setup-node@v1
        with:
          node-version: "14.x"

      - uses: actions/cache@v3
        id: node-modules-cache
        with:
          path: node_modules
          key: ${{ runner.os }}-node-modules-${{ hashFiles('yarn.lock') }}-prisma-${{ hashfiles('**/schema.prisma') }}

      - if: steps.node-modules-cache.outputs.cache-hit != 'true'
        run: |
          yarn install --frozen-lockfile
          yarn prisma:generate

      - run: yarn lint
      - run: yarn check-ts

```

Error:
```
Run yarn install --frozen-lockfile
yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning " > @tsed/schema@6.1[16](https://github.com/HoareLea/LivingLabAPI_v2/runs/7328608550?check_suite_focus=true#step:5:17).5" has unmet peer dependency "@tsed/openspec@^6.116.5".
warning " > @tsed/platform-express@6.116.5" has unmet peer dependency "@tsed/openspec@^6.116.5".
warning " > @tsed/platform-express@6.116.5" has unmet peer dependency "multer@^1.4.5-lts.1".
warning " > @tsed/prisma@6.116.5" has incorrect peer dependency "@prisma/client@^2 || ^3".
warning " > @tsed/cli-plugin-typegraphql@3.22.4" has unmet peer dependency "@tsed/cli@^3.22.4".
warning " > @tsed/cli-plugin-typegraphql@3.22.4" has unmet peer dependency "@tsed/cli-core@^3.22.4".
[4/4] Building fresh packages...
$ is-ci || husky install
Done in 42.89s.
yarn run v1.22.[19](https://github.com/HoareLea/LivingLabAPI_v2/runs/7328608550?check_suite_focus=true#step:5:20)
$ npx prisma generate
Prisma schema loaded from prisma/schema.prisma
Error: 
✔ Generated Prisma Client (4.0.0 | library) to ./node_modules/@prisma/client in 168ms
TypeError: fs_1.promises.cp is not a function
    at Object.generate [as onGenerate] (/home/runner/work/LivingLabAPI_v2/LivingLabAPI_v2/node_modules/@tsed/prisma/src/cli/prismaGenerator.ts:35:19)
    at async LineStream.<anonymous> (/home/runner/work/LivingLabAPI_v2/LivingLabAPI_v2/node_modules/@prisma/generator-helper/dist/generatorHandler.js:33:[24](https://github.com/HoareLea/LivingLabAPI_v2/runs/7328608550?check_suite_focus=true#step:5:25))
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

I'm not sure if this is the place to log this issue, but it is `/@tsed/prisma/src/cli/prismaGenerator.ts:35:19` that the error is thrown.

Without setting an output folder, I can run this fine locally, and but in the github actions I get the error above.

HOWEVER, if I set the output folder in the schema.prisma file: 
```
generator tsed {
  provider = "tsed-prisma"
  output   = "./generate"
}
```

All is good?!
```
[3/4] Linking dependencies...
warning " > @tsed/schema@6.116.5" has unmet peer dependency "@tsed/openspec@^6.116.5".
warning " > @tsed/platform-express@6.116.5" has unmet peer dependency "@tsed/openspec@^6.116.5".
warning " > @tsed/platform-express@6.116.5" has unmet peer dependency "multer@^1.4.5-lts.1".
warning " > @tsed/prisma@6.116.5" has incorrect peer dependency "@prisma/client@^2 || ^3".
warning " > @tsed/cli-plugin-typegraphql@3.22.4" has unmet peer dependency "@tsed/cli@^3.22.4".
warning " > @tsed/cli-plugin-typegraphql@3.22.4" has unmet peer dependency "@tsed/cli-core@^3.22.4".
[4/4] Building fresh packages...
$ is-ci || husky install
Done in 31.35s.
yarn run v1.22.19
$ npx prisma generate
Prisma schema loaded from prisma/schema.prisma
✔ Generated Prisma Client (4.0.0 | library) to ./node_modules/@prisma/client in [10](https://github.com/HoareLea/LivingLabAPI_v2/runs/7329089283?check_suite_focus=true#step:5:10)6ms
✔ Generated Ts.ED integration to ./prisma/generate in 327ms
You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client
\```
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()
\```
Done in 7.83s.
```

Could anyone shed a light on what's going on here?
## Acceptance criteria

- [ ] Runs in Github actions

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -76,25 +76,12 @@ yarn test
 npm run test
 ```
 
-### Gflow (optional)
-
-[Gflow](https://www.npmjs.com/package/gflow) is a command line tool to help developers with the Git process used in Ts.ED.
-
-Gflow helps you create a branch from production, rebase and run the tests before pushing your branch on your remote repository.
-
-```bash
-npm install -g gflow
-```
-
 ### Start a feature branch
 
 ```bash
 git fetch
 git checkout --no-track -b feat-branch-name origin/production  # !IMPORTANT
 yarn
-
-## OR
-gflow new feat name_of_feat
 ```
 
 ### Commit & Push a feature
@@ -112,9 +99,6 @@ npm run test
 git fetch
 git rebase origin/production
 git push -f
-
-# OR using gflow (run fetch, rebase and push for you)
-gflow push
 ```
 
 When your feature is ready to review, you can open a PR on Ts.ED github.

--- a/packages/orm/prisma/coverage.json
+++ b/packages/orm/prisma/coverage.json
@@ -1,0 +1,6 @@
+{
+  "statements": 98.16,
+  "branches": 91.46,
+  "functions": 95.38,
+  "lines": 98.15
+}

--- a/packages/orm/prisma/jest.config.js
+++ b/packages/orm/prisma/jest.config.js
@@ -6,11 +6,6 @@
 module.exports = {
   ...require("@tsed/jest-config")(__dirname, "prisma"),
   coverageThreshold: {
-    global: {
-      branches: 87.22,
-      functions: 92.42,
-      lines: 96.25,
-      statements: 92.42
-    }
+    global: require("./coverage.json")
   }
 };

--- a/packages/orm/prisma/package.json
+++ b/packages/orm/prisma/package.json
@@ -28,6 +28,7 @@
     "@prisma/sdk": "^4.0.0",
     "change-case": "^4.1.2",
     "pluralize": "^8.0.0",
+    "fs-extra": "^10.1.0",
     "ts-morph": "^12.0.0",
     "tslib": "2.4.0"
   },

--- a/packages/orm/prisma/src/cli/prismaGenerator.ts
+++ b/packages/orm/prisma/src/cli/prismaGenerator.ts
@@ -1,6 +1,6 @@
 import {GeneratorOptions} from "@prisma/generator-helper";
 import {parseEnvValue} from "@prisma/sdk";
-import {promises as asyncFs} from "fs";
+import fs from "fs-extra";
 import {generateCode} from "../generator/generateCode";
 import removeDir from "../generator/utils/removeDir";
 import path, {join} from "path";
@@ -18,7 +18,7 @@ function toUnixPath(maybeWindowsPath: string) {
 
 export async function generate(options: GeneratorOptions) {
   const outputDir = parseEnvValue(options.generator.output!);
-  await asyncFs.mkdir(outputDir, {recursive: true});
+  await fs.mkdir(outputDir, {recursive: true});
   await removeDir(outputDir, true);
 
   const generatorConfig = options.generator.config;
@@ -32,9 +32,9 @@ export async function generate(options: GeneratorOptions) {
   });
 
   if (outputDir === defaultOutput) {
-    await asyncFs.cp(join(packageDir, "scripts", "backup-index.cjs.js"), join(packageDir, "lib", "cjs", "index.js"));
-    await asyncFs.cp(join(packageDir, "scripts", "backup-index.esm.js"), join(packageDir, "lib", "esm", "index.js"));
-    await asyncFs.cp(join(packageDir, "scripts", "backup-index.d.ts"), join(packageDir, "lib", "types", "index.d.ts"));
+    await fs.copy(join(packageDir, "scripts", "backup-index.cjs.js"), join(packageDir, "lib", "cjs", "index.js"));
+    await fs.copy(join(packageDir, "scripts", "backup-index.esm.js"), join(packageDir, "lib", "esm", "index.js"));
+    await fs.copy(join(packageDir, "scripts", "backup-index.d.ts"), join(packageDir, "lib", "types", "index.d.ts"));
   }
 
   return "";

